packages/core-data/src/hooks/use-entity-record.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import { useQuerySelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as coreStore } from '../';
+import { INACTIVE, IDLE, SUCCESS, ERROR, RESOLVING } from './constants';
+
+export default function useEntityRecord(
+	kind,
+	type,
+	id,
+	options = { runIf: true }
```

Comment:
It looks like a special case that could be handled outside the hook. Can you explain the rationale behind `runIf` and how do you envision its application?